src/modules/comments/commentsCounter.test.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import commentsCounter from './commentsCounter.js';
+
+describe('Test1', () => {
+  document.body.innerHTML = `
+    <h2 id='comments__title'></h2>
+    <ul id="comment__dates"></ul>
+  `;
+  commentsCounter();
+  const title = document.getElementById('comments__title');
+  const container = document.getElementById('comment__dates');
+  const comment = document.createElement('li');
+  comment.innerHTML = 'Comment';
+  test("Checking the title when there aren't elements", () => {
+    expect(title.innerHTML).toStrictEqual('Comments (0)');
+  });
+
+  test('Checking the title if I add an element', () => {
+    container.appendChild(comment);
+    commentsCounter();
+    expect(title.innerHTML).toStrictEqual('Comments (1)');
+  });
+  test('Checking the title if I add more elements', () => {
+    container.innerHTML += `
+    <li>Comment</li>
+    <li>Comment</li>
+    <li>Comment</li>
+    <li>Comment</li>
+    <li>Comment</li>
+    <li>Comment</li>
+  `;
+    commentsCounter();
+    expect(title.innerHTML).toStrictEqual('Comments (7)');
+  });
+});
```

Comment:
- [ ] Kindly make sure that all tests pass successfully.
![image](https://user-images.githubusercontent.com/62290545/218139980-c06330d5-8452-48ee-971c-cbae3cc1d7c9.png)